sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1040,9 +1040,12 @@ def deflection(self):
                     conditions.append(((prev_def + deflection_value), args[i][1]))
                     prev_def = deflection_value.subs(x, args[i][1].args[1])
                 return Piecewise(*conditions)
-            return S(1)/(E*I)*integrate(integrate(self.bending_moment(), x), x)
+            C3 = Symbol('C3')
+            C4 = Symbol('C4')
```

Comment:
We need a way for the user to set this symbol or access it. So either an optional argurment or an attribute, or both. They should be able to set the base character(s) or symbol or pass in a generator. SymPy has a generator for creating sequential symbols. cse uses it.